programs/survival/lodestone_chunk_loader.sc

```diff
@@ -0,0 +1,53 @@
+// Lodestones are Loadstones aka chunk loaders.
+// When a player clicks on a lodestone while sneaking it marks that chunk to be force loaded
+// and if they click on it again the unmarks that chunk.
+// Breaking the loadstone also unmarks the chunk.
+
+// Note that with this script players can unmark forceload chunks set by admins.
+
+global_chunk_regex = '\\[[\\d,\\s]+\\]';
+
+_block_forceload_chunk(block, mode) -> (
+    pos = pos(block);
+    return( in_dimension(block, run(str('/forceload %s %d %d', mode, pos:0, pos:2 ))) );
```

Comment:
do you need this exlicit return statement? It doesn't hinder performance, but just checking.

Reply:
That return statement is removed by the compiler, no point hurting readability if they like to have it.

Reply:
sometimes my brain just wants to see the return.